<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressesWidget</class>
 <widget class="QWidget" name="AddressesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="container">
     <property name="styleSheet">
      <string notr="true">background-color:white;
border:none;
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,1,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>16</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">#groupBox {
	border:0;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btnMyAddresses">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    color: #bababa;
    font-size:20px;
    font-weight:light;
    background-color:transparent;
    border:0;
    padding:10px 0px 10px 0px;
    background-image: none ;
    background-repeat:no-repeat;
    background-position: center bottom;
}

QPushButton:checked {
    background-image: url(&quot;:/icons/ic-title&quot;) ;
    background-repeat:no-repeat;
    background-size:6px;
    background-position: center bottom;
    color: #105aef;
}

QPushButton:hover {
    color: #707070;
}

QPushButton:unchecked {
    background-image: none ;
    color: #bababa;
}</string>
             </property>
             <property name="text">
              <string>My addresses</string>
             </property>
             <property name="iconSize">
              <size>
               <width>6</width>
               <height>6</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnContacts">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    color: #bababa;
    font-size:20px;
    font-weight:light;
    background-color:transparent;
    border:0;
    padding:10 0 10 0;
    background-image: none ;
    background-repeat:no-repeat;
    background-position: center bottom;
}

QPushButton:checked {
    background-image: url(&quot;:/icons/ic-title&quot;) ;
    background-repeat:no-repeat;
    background-position: center bottom;
    color: #105aef;
}

QPushButton:hover {
    color: #707070;
}

QPushButton:unchecked {
    background-image: none ;
    color: #bababa;
}</string>
             </property>
             <property name="text">
              <string>Contacts</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnMiningAddress">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #707070;
font-size:18px;
font-weight:light;	
background-color:transparent;
border:none;
padding:4px 0px 4px 4px;</string>
          </property>
          <property name="text">
           <string>New mining address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMiningAddress2">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:transparent;
border:none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../veil.qrc">
            <normaloff>:/icons/ic-mined-png</normaloff>:/icons/ic-mined-png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSplit">
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#bababa;
margin-top:12px;
margin-bottom:12px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #707070;
font-size:18px;
font-weight:light;	
background-color:transparent;
border:none;
padding:4px 0px 4px 6px;</string>
          </property>
          <property name="text">
           <string>Add contact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddIcon">
          <property name="styleSheet">
           <string notr="true">background-color:transparent;
border:none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../veil.qrc">
            <normaloff>:/icons/ic-add-account-png</normaloff>:/icons/ic-add-account-png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QFrame" name="empty">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../veil.qrc">:/icons/img-empty-addresses</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="containerEmpty">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,3">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>24</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#707070;&quot;&gt;No available addresses to show&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listAddresses">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QListView { 
background: transparent;
 }

/* Works for both QListView and QListWidget */
QListView::item {
    /* Won't work without borders set */
    border: 0px;

    padding: 10px;
}

/* For icon only */
QListView::icon {
    left: 10px;
}

/* For text only */
QListView::text {
    left: 10px;
    font-size:13px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="movement">
         <enum>QListView::Free</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="layoutMode">
         <enum>QListView::SinglePass</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listContacts">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QListView { 
background: transparent;
 }

/* Works for both QListView and QListWidget */
QListView::item {
    /* Won't work without borders set */
    border: 0px;

    padding: 10px;
}

/* For icon only */
QListView::icon {
    left: 10px;
}

/* For text only */
QListView::text {
    left: 10px;
    font-size:13px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="movement">
         <enum>QListView::Free</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../veil.qrc"/>
 </resources>
 <connections/>
</ui>
